integration/test-integration/src/test/resources/testng-cc-with-apim.xml

```diff
@@ -70,13 +76,6 @@
             <class name="org.wso2.choreo.connect.tests.testcases.withapim.EnforcerAPITestCase"/>
         </classes>
     </test>
-<!--    <test name="apis-apps-subs-received-after-apim-restart" parallel="false">-->
-<!--        <classes>-->
-<!--            <class name="org.wso2.choreo.connect.tests.setup.withapim.ApimRestartExecutor"/>-->
-<!--            <class name="org.wso2.choreo.connect.tests.testcases.withapim.ExistingApiTestCase"/>-->
```

Comment:
did this test move anywhere?

Reply:
This was commented previously because of an issue it introduced. We do this now using the restartexecutor